inc/inventorycommon.class.php

```diff
@@ -139,8 +154,8 @@ function importPorts($itemtype, $a_inventory, $items_id) {
             $pfNetworkPort->update($a_port);
          } else {
             // Update port
-            $networkports_id = $a_ports_DB['id'];
-            $a_port['id'] = $a_ports_DB['id'];
+            $networkports_id = $networkPort->getID();
```

Comment:
Maybe it should be better not to use `getID()`; we've faced some strange issues using it.

Reply:
done